src/components/rangeslider/defaults.js

```diff
@@ -38,6 +44,31 @@ module.exports = function handleDefaults(layoutIn, layoutOut, axName) {
     coerce('autorange', !axOut.isValidRange(containerIn.range));
     coerce('range');
 
+    var subplots = layoutOut._subplots;
+    if(subplots) {
+        var yIds = subplots.yaxis;
+        var yNames = Lib.simpleMap(yIds, axisIds.id2name);
+        for(var i = 0; i < yNames.length; i++) {
+            var yName = yNames[i];
+            if(!containerIn[yName]) {
+                containerIn[yName] = {};
+            }
+            if(!containerOut[yName]) {
+                containerOut[yName] = {};
+            }
```

Comment:
Do we *need* to push `{}` back to `containerIn`? I know we have many examples to the contrary 🙈 but we try not to modify the inputs any more than we have to. Also at this point we know `containerOut[yName]` doesn't exist yet. So ideally this section would look something like (would require changing `coerceRange` but should actually make it simpler):
```js
rangeContainerIn = containerIn[yName] || {};
rangeContainerOut = containerOut[yName] = {};
```

Reply:
Indeed, it is not required anymore (it was in one of the first version).